VehicleCode/TestingAndCalibration/ReceiveTesting/src/Car.cpp

```diff
@@ -360,19 +358,6 @@ int Car::tempLength(const int &maxNumber) {
 
 
 
-
```

Comment:
I'd like to leave the empty space, it helps me find different sections when editing

Reply:
Works for me